### PR TITLE
chore(docs): Set up Charmhub docs and update tutorial.

### PR DESCRIPTION
**Pull request by alithethird**
Applicable specs: ISD-2284, ISD-2303, ISD-2317

### Overview

Created the Discourse topics for documents and tied it up to Charmhub.
### Checklist

- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

[ISD-2284]: https://warthogs.atlassian.net/browse/ISD-2284?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ISD-2303]: https://warthogs.atlassian.net/browse/ISD-2303?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[ISD-2317]: https://warthogs.atlassian.net/browse/ISD-2317?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by arturo-seijas — DISMISSED** [review]

**Review by javierdelapuente — DISMISSED** [review]

File: charm/docs/reference/configurations.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Integrations
+# Configurations
 
-See [Integrations](https://charmhub.io/httprequest-lego-provider/integrations).
+See [Configurations](https://charmhub.io/httprequest-lego-provider/configure).
```
Comment (erinecon) [inline charm/docs/reference/configurations.md:3]: Below this line, could you add:

> Read more about configurations in the Juju docs: [Configuration](https://juju.is/docs/juju/configuration)

File: charm/docs/tutorial/getting-started.md [hunk 2]
```diff
@@ -7,12 +7,23 @@
...
-
-- Juju 3 installed.
-- Juju controller and model created.
+- A working station, e.g., a laptop, with amd64 architecture.
+- Juju 3 installed and bootstrapped to a MicroK8s controller. You can accomplish this process by following this guide: [Set up / Tear down your test environment](https://juju.is/docs/juju/set-up--tear-down-your-test-environment)
+- NGINX Ingress Controller. If you're using [MicroK8s](https://microk8s.io/), this can be done by running the command `microk8s enable ingress`. For more details, see [Addon: Ingress](https://microk8s.io/docs/addon-ingress).
 
 For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
+:warning: When using a Multipass VM, make sure to replace `127.0.0.1` IP addresses with the
```
Comment (erinecon) [inline charm/docs/tutorial/getting-started.md:16]: I think that the :warning: and "Shell into the Multipass VM" section should go after the "Steps" header

**Review by erinecon — COMMENTED** [review]
Hi, thanks for updating the docs! 

Could you also update the `docs/reference/actions.md` file to add the following line at the end:
> Read more about actions in the Juju docs: [Action](https://juju.is/docs/juju/action)

File: charm/docs/index.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Documentation is an important part of this project, and we take the same open-source approach to the documentation as the code. As such, we welcome community contributions, suggestions and constructive feedback on our documentation. Our documentation is hosted on the [Charmhub forum](https://discourse.charmhub.io/) to enable easy collaboration. Please use the "Help us improve this documentation" links on each documentation page to either directly change something you see that's wrong, or ask a question, or make a suggestion about a potential change via the comments section.
+
+If there's a particular area of documentation that you'd like to see that's missing, please [file a bug](https://github.com/canonical/httprequest-lego-provider/issues).
+
+## Project and community
+
+HTTPRequest Lego Provider is an open-source project that welcomes community contributions, suggestions, fixes and constructive feedback.
+- [Read our Code of Conduct](https://ubuntu.com/community/code-of-conduct)
+- [Get support](https://discourse.charmhub.io/tag/mattermost)
```
Comment (erinecon) [inline charm/docs/index.md:38]: I think this line needs to be updated?

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by erinecon — APPROVED** [review]
Looks good -- thanks so much!

**Review by javierdelapuente — APPROVED** [review]

**Pull request merged**